base/applications/mspaint/winproc.cpp

```diff
@@ -575,6 +577,44 @@ LRESULT CMainWindow::OnSysColorChange(UINT nMsg, WPARAM wParam, LPARAM lParam, B
     return 0;
 }
 
+typedef ULONG (WINAPI *FN_MAPISendMailW)(LHANDLE, ULONG_PTR, lpMapiMessageW, FLAGS, ULONG);
+
+ULONG CMainWindow::OpenMailerWithAttachmentW(LPCWSTR pszFileName)
+{
+    if (!PathFileExists(pszFileName))
+    {
+        ::ExpandEnvironmentStrings(_T("%TEMP%\\Image.png"), g_szTempFile, _countof(g_szTempFile));
```

Comment:
Why doesn't mspaint use explicit unicode functions and strings by now?

Reply:
Someone wants to port to Win9x?

Reply:
none wants to do that. Win9x already has an mspaint. Also, the fact ours use gdiplus makes me think it wouldn't be usable on win9x.